iamctl/pkg/identityProviders/import.go

```diff
@@ -84,22 +87,41 @@ func importIdp(idpId string, importFilePath string) error {
 	// Replace keyword placeholders in the local file according to the keyword mappings added in configs.
 	fileInfo := utils.GetFileInfo(importFilePath)
 	idpKeywordMapping := getIdpKeywordMapping(fileInfo.ResourceName)
-	modifiedFileData := utils.ReplaceKeywords(string(fileBytes), idpKeywordMapping)
+	fileDataWithReplacedKeywords := utils.ReplaceKeywords(string(fileBytes), idpKeywordMapping)
+	modifiedFileData := utils.RemoveSecretAsterisks(fileDataWithReplacedKeywords)
```

Comment:
We don't need to do this removal here. This should be done only for applications